src/main/java/frc/robot/RobotContainer.java

```diff
@@ -147,8 +147,14 @@ private void configureBindings() {
 
     backupController.a().whileTrue(CompoundCommands.armToReef(reef_level));
 
-    controller.rightTrigger(0.25).whileTrue(intake.ejectCommand());
-    controller.leftTrigger(0.25).whileTrue(intake.intakeCommand());
+    controller
+        .leftTrigger(0.25)
+        .onTrue(intake.runIntakeCommand(() -> 3).alongWith(CompoundCommands.armToSource()))
+        .onFalse(intake.runIntakeCommand(() -> 0.5).alongWith(CompoundCommands.armToStow()));
```

Comment:
Another option is to make armToStow() the default command for arm. Not sure if it's better yet.